types/eleventy/collection-item.d.ts

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Is eleventy on definitely typed? Just curious if there's a place where this stuff is maintained so we don't have to stay in sync with eleventy when it revs.

Reply:
Unfortunately not... It wasn't super difficult to generate this though!